components/dashboard/sections/expenses/HostCreateExpenseModal.tsx

```diff
@@ -479,31 +477,43 @@ export const HostCreateExpenseModal = ({
                       label={
                         values.type === ExpenseType.RECEIPT ? (
                           <FormattedMessage defaultMessage="Receipt" id="Expense.Receipt" />
+                        ) : values.type === ExpenseType.CHARGE ? (
+                          <FormattedMessage defaultMessage="Receipt" id="Expense.Receipt" />
```

Comment:
If using the same value, I think we could combine with the condition check above.